src/main/java/org/triplea/client/launch/LaunchScreen.java

```diff
@@ -0,0 +1,654 @@
+    final Dimension size = frame.getSize();
+    frame.getContentPane().removeAll();
+    frame.getContentPane().add(component);
+    frame.setSize(size);
+    frame.pack();
+    frame.setSize(size);
+  }
+
+
+  private JPanel buttonPanel() {
+    return buttonPanel(null);
+  }
+
+  private JPanel buttonPanel(final Runnable backAction) {
+    return buttonPanel(backAction, null);
+  }
+
+  private JPanel buttonPanel(final Runnable backAction, final Runnable playAction) {
+    return JPanelBuilder.builder()
+        .borderEtched()
+        .flowLayout()
+        .add(JButtonBuilder.builder()
+            .biggerFont()
+            .title("Play")
+            .actionListener(playAction == null ? () -> {
+            } : playAction)
+            .enabled(playAction != null)
+            .build())
+        .add(Box.createHorizontalStrut(50))
+        .add(JButtonBuilder.builder()
+            .title("Back")
+            .actionListener(backAction == null ? () -> {
+            } : backAction)
+            .enabled(backAction != null)
+            .build())
+        .add(Box.createHorizontalStrut(50))
+        .add(JButtonBuilder.builder()
+            .title("Quit")
+            .actionListener(() -> {
+              frame.dispose();
+            })
+            .build())
+        .add(Box.createVerticalGlue())
+        .build();
+  }
+
+
+
+  private void showMoreOptions() {
+    replaceContentPaneContents(
+        JPanelBuilder.builder()
+            .addCenter(
+                JPanelBuilder.builder()
+                    .flowLayout()
+                    .add(
+                        JPanelBuilder.builder()
+                            .verticalBoxLayout()
+                            .add(Box.createVerticalStrut(40))
+                            .add(JButtonBuilder.builder()
+                                .title("Settings")
+                                .actionListener(ClientSetting::showSettingsWindow)
+                                .build())
+                            .add(Box.createVerticalStrut(40))
+                            .add(JButtonBuilder.builder()
+                                .title("Map Tools")
+                                .actionListener(() -> ProcessRunnerUtil.runClass(MapCreator.class))
+                                .build())
+                            .add(Box.createVerticalGlue())
+                            .build())
+                    .build())
+            .addSouth(buttonPanel(INSTANCE::showInitialScreen))
+            .build());
+  }
+
+  private void showMultiplayer() {
+    replaceContentPaneContents(
+        JPanelBuilder.builder()
+            .addCenter(
+                JPanelBuilder.builder()
+                    .flowLayout()
+                    .add(
+                        JPanelBuilder.builder()
+                            .verticalBoxLayout()
+                            .add(Box.createVerticalStrut(40))
+                            .add(JButtonBuilder.builder()
```

Comment:
Going back and looking at this while I'm awake :smile:, I think what triggered my comment was a few lines down in the `actionListener()` call for the "Play Online" button.  The fact that the body of this method starts in column 35 means the formatter may do some pretty aggressive line wrapping.

If this particular lambda were extracted to a named method, it would have looked similar to the other code in this file, and I probably wouldn't have given the indentation a second thought. :smile: